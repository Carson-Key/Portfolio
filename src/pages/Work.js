// Components
import Container from '../components/Container'
import WorkCard from '../ui/WorkCard'
// Assets
import LogoGuardianGroup from '../assets/Logo-GuardianGroup.png'
import LogoCBTNuggets from '../assets/Logo-CBTNuggets.png'
import LogoOSU from '../assets/Logo-OSU.png'
import LogoAHC from '../assets/Logo-AHC.png'

const Work = () => {
    return (
		<Container className="flex flex-wrap justify-evenly">
			<WorkCard
				title="CBT Nuggets"
				imageSrc={LogoCBTNuggets}
				imageAlt="Two lower case g's made of three golden circles"
				description="At this position my main task was developing a bug filled website to use in the QA engineer interview process. After finishing the project early, I moved onto creating a dashboard to display automated test statistics."
				role="Intern"
				years={{from: "Jun 2018", to: "Sep 2018"}}
				projects={[{name: "Bug Hunt", id: "#cbtnuggetsbughunt"}, {name: "QA Statistics Dashboard", id: "#cbtnuggetsqadashboard"}]}
			/>
		</Container>
	)
}

export default Work
