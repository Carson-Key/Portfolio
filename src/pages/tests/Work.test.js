// Packages
import renderer from 'react-test-renderer'
// Pages
import Work from '../Work'

describe('Work Page', () => {
    it('Should Render Work page', () => {
        const work = renderer.create(
            <Work />
        ).toJSON()
        expect(work).toMatchSnapshot()
    })
})