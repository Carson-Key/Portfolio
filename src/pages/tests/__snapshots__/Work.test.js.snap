// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Work Page Should Render Work page 1`] = `
<main>
  <p>
    Work
  </p>
</main>
`;
